examples/ansible/kubeone/galaxy.yml

```diff
@@ -0,0 +1,12 @@
+namespace: kubermatic
+name: kubeone
+version: "0.10.0-alpha.0"
+readme: README.md
+authors:
+  - "Alexander Sowitzki <alexander@loodse.com>"
```

Comment:
Legal question (who can we ask legal questions?): should this be `The KubeOne Authors` and some public email address, like we put in the license and license headers?

Reply:
I see the point, would be nice. But: Do we have one? Do we want one?